src/MyPlot/task/FillPlotTask.php

```diff
@@ -104,6 +104,6 @@ public function onRun(int $currentTick) : void {
 				}
 			}
 		}
-		$this->plugin->getLogger()->debug("Plot Clear task completed at {$this->plotBeginPos->x};{$this->plotBeginPos->z}");
+		$this->plugin->getLogger()->debug("Plot Fill task completed at {$this->plotBeginPos->x};{$this->plotBeginPos->z}");
 	}
-}
```

Comment:
Line endings...

Reply:
Looks like you can't even use Githubs Editor for a typo fix...